cmd/version.go

```diff
@@ -20,7 +20,7 @@ func initVersionCmd() {
 	var versionCmd = &cobra.Command{
 		Use:   "version",
 		Short: "Print the version",
-		Long:  `Display gscloud version information.`,
+		Long:  `Print version information.`,
```

Comment:
You are right Print makes more sense.
What about `Print current gscloud version.` ?

Reply:
Nahh. The less said, the better.